### PR TITLE
Feature#3 database model

### LINKED ISSUE #3: Database Model
# REQUIREMENTS
Create database model for project and code tables to have ready database

# Acceptance Criteria
- [x] Models complete
- [x] Tables created.

### PR DESCRIPTION
**Pull request by SampabloC**
# Description
- Create relationship entity diagram based on requirements for project.
- Created tables based on diagram.
- Generate schema file.

Resolve #3

  ## Screenshots 
imagen

### CODE REVIEW

**Review by YilKanda — DISMISSED** [review]
Just as a suggestion, remember that in the normalization they tell us that the tables must be distinctive, the user table and the psychologist table have some similar fields (name, email, password, age, gender), therefore, the psychologist table it should depend on user, so it doesn't have those same fields.

**Review by jairolan — DISMISSED** [review]

File: db/migrate/20210806041307_appointments.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Appointments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :appointments do |t|
+      t.date :date
+      t.time :time
+      t.boolean :pay
```
Comment (Ilyeo) [inline db/migrate/20210806041307_appointments.rb:6]: minor suggestion here

```suggestion
      t.boolean :pay, null: false, default: false
```
[Avoid the Three-state Boolean Problem](https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem)

Comment (SampabloC) [reply db/migrate/20210806041307_appointments.rb]: Thanks, I corrected that on a new commit

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by juliocabrera820 — APPROVED** [review]

**Review by brayandiazc — APPROVED** [review]
They could use a single user table and define the user type in the record field.
But they can continue as they are if they feel comfortable.

**Pull request merged**